Makefile

```diff
@@ -260,27 +274,27 @@ define gen-bash-completions-help
 # Helper-target to produce Bash-completions for tools (tools, examples, tests)
 #
 # NOTE: This target requires a bunch of things: binaries must be built and
-# residing in 'builddir/tools' etc. AND installed on the system. Also, the find
+# residing in '$(BUILD_DIR)/tools' etc. AND installed on the system. Also, the find
```

Comment:
I really do prefer that unrelated changes, such as these fixing the hardcoded builddir, would go into a separate commit.